packages/ember-simple-auth/addon/session-stores/adaptive.js

```diff
@@ -102,8 +100,21 @@ export default Base.extend({
   _cookieExpirationTime: null,
   cookieExpirationTime: proxyToInternalStore(),
 
+  _sameSite: null,
+  sameSite: proxyToInternalStore(),
+
   _cookies: service('cookies'),
 
+  _isLocalStorageAvailable: computed({
```

Comment:
Could we not keep the regular property as it's used in [line 115](https://github.com/simplabs/ember-simple-auth/pull/2312/files#diff-db3487bdb1f8c84714dedc96d1cce936b66f804e8a3c58ebe9d88dfb883e50deL115). Once we move away from Ember objects/computeds, that's how we'd have to do it anyway?

Reply:
Not sure at the moment if this is needed - will check once I pick up the PR again, although I'm pretty sure I needed that to correctly test initialization of services

Reply:
Unfortunately we need this to be here, in tests to tell the adaptive store that localStorage is not available we use `Adaptive.extend({ _isLocalStorageAvailable: false })` which would be overwritten on `init`.

I see that the previous init was doing `hasOwnProperty` as well, but that was working only because we were creating the stores manually passing properties to `create`, but `extend` extends the prototype and `_isLocalStorage` is no longer an 'own property'.